README.md

```diff
@@ -165,4 +165,4 @@ Before submitting a pull request, please make sure the following is done:
 3. Ensure the test suite passes (`npm test`).
 
 ###Style Guide
```

Comment:
That also applies for the ##How to Contribute and ###Submitting a Pull Request, but I don't know why I can't comment those lines.